src/player.js

```diff
@@ -517,12 +517,12 @@ export default class Player extends FakeEventTarget {
 
   /**
    * Start/resume playback.
-   * @param {boolean} programmatic - indicates the play call was not user gestured and should be handled like auto play.
+   * @param {PKPlayOptionsObject} playOptions - additional options to control the play. i.e. programmatic true indicates the play call was not initiated by a user gesture and should be handled like auto play.
```

Comment:
see example on hot to describe object properties with jsdoc https://stackoverflow.com/questions/6460604/how-to-describe-object-arguments-in-jsdoc

Reply:
done